torchvision/csrc/io/decoder/stream.cpp

```diff
@@ -1,5 +1,8 @@
 #include "stream.h"
 #include <c10/util/Logging.h>
+#include <stdio.h>
+#include <string.h>
+#include <torch/torch.h>
```

Comment:
We try to keep the header declarations as specific as possible. I think this can be replaced with
```suggestion
#include <ATen/Parallel.h>
```